src/deleteTask.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line import/no-cycle
+import { createList } from './addtask.js';
+
+export default function deleteTask(items) {
+  const deleteButtons = document.querySelectorAll('i');
+  deleteButtons.forEach((button, index) => {
+    button.addEventListener('click', () => {
+      items.splice(index, 1);
+      items.forEach((item, i) => {
+        item.index = (i + 1);
+      });
+      localStorage.localData = JSON.stringify(items);
+      createList(items);
+    });
+  });
+}
```

Comment:
- [ ] The button to clear all completed tasks is not working, let's add a function for the "Clear all completed" button using the `Array.filter()` method.

Reply:
Hi @CRClothier , you have made nice progress making the `clear all completed` button work. But it lacks one required behavior. When tasks are deleted the indexes of the remaining tasks should be re-adjusted to be a consecutive integer. 

![Screenshot from 2021-12-11 16-07-14](https://user-images.githubusercontent.com/64894927/145677791-17ad34a7-052a-44a5-ab31-672cda922c7a.png)